nixos/modules/config/networking.nix

```diff
@@ -193,6 +193,10 @@ in
           cat ${escapeShellArgs cfg.hostFiles} > $out
         '';
 
+        # /etc/netgroup: Network-wide groups.
+        netgroup.text = ''
```

Comment:
We might want this to be an `mkDefault`, so it's easier to be overridden, if someone actually wants to add something else in there.

Reply:
It makes sense, I will do so. Thank you for reviewing!

Reply:
Done.